src/App.js

```diff
@@ -37,19 +39,13 @@ function App() {
 
         <Box p="6">
           <Box display="flex" alignItems="baseline">
-            <Badge borderRadius="full" px="2" colorScheme="teal">
-              New
-            </Badge>
-            <Box
-              color="gray.500"
-              fontWeight="semibold"
-              letterSpacing="wide"
-              fontSize="xs"
-              textTransform="uppercase"
-              ml="2"
-            >
-              {property.beds} beds &bull; {property.baths} baths
-            </Box>
+          <StarIcon p={0.5} w={5} h={5}/>
+          <ChatIcon p={0.5} w={5} h={5}/>
+          <ExternalLinkIcon p={0.5} w={5} h={5}/>
+          <Spacer/>
+          <DownloadIcon p={0.5} w={5} h={5}/>
```

Comment:
Great, it seems now it's a good time for extracting some components, this one is getting too big don't you think?